### ISSUE TITLE
Allow "cargo install gluon" to work

### ISSUE THREAD

**Issue by salpalvv** [gluon-lang/gluon#257]
When I run `cargo install gluon` to add the repl binary to my path and execute `gluon -i` I get: 
```
$ gluon -i
No such file or directory (os error 2)
```

As I understand it, it's looking for `.glu` files from a relative directory [here](https://github.com/gluon-lang/gluon/blob/master/src/repl.rs#L179). 
```rust
compiler.load_file_async(vm, "std/repl.glu").sync_or_error()?;
```

I would like to add to my vim plugin the ability to execute the repl from within vim on snippets of code or the whole file. 
I'm not sure where the `.glu` files would be required to exist for them to be picked up.

**Comment by Marwes** [comment]
I guess the way to fix this is to add a build script which copies all the files in `std/` to the output directory. Then `src/repl.rs` can be modified to load the file relative to the executable.

**Comment by salpalvv** [comment]
from the `cargo install --help` docs:

```
This command manages Cargo's local set of installed binary crates. Only packages
which have [[bin]] targets can be installed, and all binaries are installed into
the installation root's `bin` folder. The installation root is determined, in
order of precedence, by `--root`, `$CARGO_INSTALL_ROOT`, the `install.root`
configuration key, and finally the home directory (which is either
`$CARGO_HOME` if set or `$HOME/.cargo` by default).
```

So, it looks like there would be multiple places to check if `std/` is dropped right beside the binary. Would it be better to have a default, well-formed location? I, personally, don't like the thought of `.glu` files sitting in my `bin/` dir.  
 
Perhaps there could be a `$GLUON_STD` or `$GLUON_LIB` env variable that's set? `build.rs` could copy `std` there and then `src/repl.rs` would check there with precedence and fall back to relative.

**Comment by Marwes** [comment]
`cargo install gluon_repl` (moved the app to its own crate) is working now since all gluon files are compiled into the program itself, avoiding the issue with directories.I think this issue can be closed for now but long term the library files should be distributed separately from the app better binary distribution should be fixed first (#194). 

GLUON_PATH can be used to specify additional directories to lookup imports.
